client/state/reducer.js

```diff
@@ -98,6 +97,7 @@ import wordads from './wordads/reducer';
 
 // Legacy reducers
 // The reducers in this list are not modularized, and are always loaded on boot.
+// Please do not add to this list. See #39261 and p4TIVU-9lM-p2 for more details.
```

Comment:
Add a lint error maybe?

Reply:
I think I've figured out a way of doing that with `no-restricted-imports`. I'll work on that in a different PR.